jmclient/jmclient/wallet_service.py

```diff
@@ -498,14 +510,26 @@ def sync_unspent(self):
         our_unspent_list = [x for x in unspent_list if (
             self.bci.is_address_labeled(x, wallet_name) or
             self.bci.is_address_labeled(x, self.EXTERNAL_WALLET_LABEL))]
-        for u in our_unspent_list:
-            if not self.is_known_addr(u['address']):
+        for utxo in our_unspent_list:
```

Comment:
I think it's *really* a stretch to argue that single letter variables are not even acceptable in simple for loops! :)

Reply:
I don't think it was that (this was a few weeks ago now and I don't remember). I think it was more about matching the name in `_add_unspent_txo` (?)